client/core/core.go

```diff
@@ -3689,6 +3713,9 @@ func (c *Core) Logout() error {
 		dc.acct.lock()
 	}
 
+	c.mtx.Lock()
+	c.loginWg = nil
```

Comment:
Oh I just realized this PR makes future Login calls (not just the ones that are waiting on concurrent ones to finish) skip over the wallet connect and `c.resolveActiveTrades` calls.

I think for 0.6 we look into a more drastic change that nukes the LoginResult so we don't even have to do `authDEX`, but for 0.5 we don't break the API and return the result the caller expects.

Reply:
I'm going to propose a different, less fundamental, change for 0.5, but we can keep investigating in this PR.

The thing about the first login attempt forever blocking subsequent logins from attempting wallet connections and `c.resolveActiveTrades` is that if a wallet failed to connect on that initial attempt, **the `trackedTrade`s using that wallet will not be loaded into the `dc.trades` map**, and they will have just vanished until they can run `c.resolveActiveTrades` again with connected wallets.  Maybe we need to rethink design, but for now I don't want to eliminate the ability to have this reattempted by calling `Login` again.  The user can try to `Logout` so that they may `Login` again, but they'll get blocked from doing that if there are other active orders.

In matrix we were discussing a `Refresh` function perhaps, or putting an an override arg to Logout.  Probably we should load all trades into the map, even if they don't have a wallet, but that will also take work and probably break things.  So these are future changes to make.